docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -425,6 +426,7 @@ Create a container
           You must use this with `memory` and make the swap value larger than `memory`.
     -   **MemoryReservation** - Memory soft limit in bytes.
     -   **KernelMemory** - Kernel memory limit in bytes.
+    -   **NanoCPUs** - CPU quota in units of 10<sup>-9</sup> CPUs.
```

Comment:
Not your fault, just reading through these, and wondering. So yes, it may need some extra information to prevent confusion